doc/interop-test-descriptions.md

```diff
@@ -509,21 +508,29 @@ Procedure:
       }
     }
     ```
+The client attaches custom metadata with the following keys and values:
+    ```
+    key: "x-grpc-test-echo-initial", value: "test_initial_metadata_value"
+    key: "x-grpc-test-echo-trailing", value: 0xababab
```

Comment:
`0xababab` isn't ASCII and so isn't valid for that key. Were you wanting it to be a binary key?